prism-agent/service/server/src/main/scala/io/iohk/atala/pollux/credentialschema/SchemaRegistryEndpoints.scala

```diff
@@ -128,6 +130,26 @@ object SchemaRegistryEndpoints {
       )
       .tag("Schema Registry")
 
+  val getRawSchemaByIdEndpoint: PublicEndpoint[
+    (RequestContext, UUID),
+    ErrorResponse,
+    Json, // changed to generic Json type
+    Any
+  ] =
+    endpoint.get
+      .in(extractFromRequest[RequestContext](RequestContext.apply))
+      .in(
+        "schema-registry" / "raw-schemas" / path[UUID]("guid").description(
```

Comment:
I would suggest using the same [approach](https://github.com/hyperledger-labs/open-enterprise-agent/blob/f77b796e8f4d0e5e421b0707bf8c12ec1daf9f6a/prism-agent/service/server/src/main/scala/io/iohk/atala/pollux/credentialdefinition/CredentialDefinitionRegistryEndpoints.scala#L101) as for credential definition and make it a "sub path" of the parent schema path, with the name of the targeted attribute as suffix. This means `"schema-registry" / "schemas" / path[UUID]("guid") / "schema"`. This feels more natural for the user, is more aligned with RESTful best practices, and is consistent with the current codebase.

Reply:
fixed